tasks/prepare-fbc-release/tests/test-prepare-fbc-release.yaml

```diff
@@ -108,3 +108,17 @@ spec:
                 cat $(workspaces.data.path)/mock_skopeo.txt
                 exit 1
               fi
+
+              # ocp version in skopeo mock-up is set to v4.12
+              ocpVersion="v4.12"
+
+              DATA_FILE="$(workspaces.data.path)/data.json"
+
+              # Read components and initial values
+              fromIndex=$(jq -r '.fbc.fromIndex' "$DATA_FILE")
```

Comment:
Doesn't the task modify the snapshot file, not data file? So this should check the values on the component in the snapshot? It looks right in the other test. Basically what I am saying is I don't see how checking values from the data file tests anything because the data file isn't changed in the task

Reply:
true. fixed!